internal/compiler/generator/cpp.rs

```diff
@@ -593,13 +593,118 @@ pub fn generate(doc: &Document) -> impl std::fmt::Display {
                 }))
             }
             #[cfg(feature = "software-renderer")]
-            crate::embedded_resources::EmbeddedResourcesKind::BitmapFontData(_) => {
-                // FIXME! TODO
+            crate::embedded_resources::EmbeddedResourcesKind::BitmapFontData(
+                crate::embedded_resources::BitmapFont {
+                    family_name,
+                    character_map,
+                    units_per_em,
+                    ascent,
+                    descent,
+                    glyphs,
+                    weight,
+                    italic,
+                },
+            ) => {
+                let family_name_var = format!("slint_embeeded_resource_{}_family_name", er.id);
+                let family_name_size = family_name.len();
                 file.declarations.push(Declaration::Var(Var {
-                    ty: "inline int".into(),
+                    ty: "inline uint8_t".into(),
```

Comment:
I'm thinking, should this not be const?

There are other place where i think we should have added a const and we didn't, btw.
If this means difference between data and rodata, that's a big difference.

Reply:
Good point! I'll have a look.